cpp/tests/strings/case_tests.cu

```diff
@@ -85,3 +86,35 @@ TEST_F(StringsCaseTest, EmptyStringsColumn)
     auto view = results->view();
     cudf::test::expect_strings_empty(results->view());
 }
+
+TEST_F(StringsCaseTest, Capitalize)
+{
+    std::vector<const char*> h_strings{ "Examples aBc", "thesé", nullptr, "ARE THE", "tést strings", "" };
+    std::vector<const char*> h_expected{ "Examples abc", "Thesé", nullptr, "Are the", "Tést strings", "" };
+
+    cudf::test::strings_column_wrapper strings( h_strings.begin(), h_strings.end(),
+        thrust::make_transform_iterator( h_strings.begin(), [] (auto str) { return str!=nullptr; }));
+    auto strings_view = cudf::strings_column_view(strings);
+
+    auto results = cudf::strings::capitalize(strings_view);
+
+    cudf::test::strings_column_wrapper expected( h_expected.begin(), h_expected.end(),
+        thrust::make_transform_iterator( h_expected.begin(), [] (auto str) { return str!=nullptr; }));
+    cudf::test::expect_columns_equal(*results,expected);
+}
+
+TEST_F(StringsCaseTest, Title)
+{
+    std::vector<const char*> h_strings{ "Examples aBc", "thesé", nullptr, "ARE THE", "tést strings", "" };
```

Comment:
As mentioned in a previous comment. 
The following string will create different sized outputs: `"SȺȺnich"`
The expected string would be `"Sⱥⱥnich"`
These could be added to the `Capitalize` test as well.

Reply:
Done.